<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsCustomProjectionDialogBase</class>
 <widget class="QDialog" name="QgsCustomProjectionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Coordinate Reference System Definition</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Define</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>You can define your own custom Coordinate Reference System (CRS) here. The definition must conform to the proj4 format for specifying a CRS.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>leName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel3_2">
        <property name="text">
         <string>Parameters</string>
        </property>
        <property name="buddy">
         <cstring>leParameters</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leParameters"/>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QToolButton" name="pbnFirst">
          <property name="text">
           <string>|&lt;</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../images/themes/default/mIconFirst.png</normaloff>../../images/themes/default/mIconFirst.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnPrevious">
          <property name="text">
           <string>&lt;</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../images/themes/default/mIconPrevious.png</normaloff>../../images/themes/default/mIconPrevious.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblRecordNo">
          <property name="text">
           <string>1 of 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnNext">
          <property name="text">
           <string>&gt;</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../images/themes/default/mIconNext.png</normaloff>../../images/themes/default/mIconNext.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnLast">
          <property name="text">
           <string>&gt;|</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../images/themes/default/mIconLast.png</normaloff>../../images/themes/default/mIconLast.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnNew">
          <property name="text">
           <string>*</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../images/themes/default/mIconNew.png</normaloff>../../images/themes/default/mIconNew.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnSave">
          <property name="text">
           <string>S</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../images/themes/default/mActionFileSave.png</normaloff>../../images/themes/default/mActionFileSave.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnDelete">
          <property name="text">
           <string>X</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../images/themes/default/mIconDelete.png</normaloff>../../images/themes/default/mIconDelete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Test</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Use the text boxes below to test the CRS definition you are creating. Enter a coordinate where both the lat/long and the transformed result are known (for example by reading off a map). Then press the calculate button to see if the CRS definition you are creating is accurate.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel3_2_2">
        <property name="text">
         <string>Parameters</string>
        </property>
        <property name="buddy">
         <cstring>leTestParameters</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="leTestParameters"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>Geographic / WGS84</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="textLabel2_3">
        <property name="text">
         <string>Destination CRS        </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>North</string>
        </property>
        <property name="buddy">
         <cstring>northWGS84</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="northWGS84">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="projectedX">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel2_2_2">
        <property name="text">
         <string>East</string>
        </property>
        <property name="buddy">
         <cstring>eastWGS84</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="eastWGS84"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="projectedY">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QPushButton" name="pbnCalculate">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leParameters</tabstop>
  <tabstop>pbnFirst</tabstop>
  <tabstop>pbnPrevious</tabstop>
  <tabstop>pbnNext</tabstop>
  <tabstop>pbnLast</tabstop>
  <tabstop>pbnNew</tabstop>
  <tabstop>pbnSave</tabstop>
  <tabstop>pbnDelete</tabstop>
  <tabstop>leTestParameters</tabstop>
  <tabstop>northWGS84</tabstop>
  <tabstop>eastWGS84</tabstop>
  <tabstop>pbnCalculate</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsCustomProjectionDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
